Doc/library/xml.rst

```diff
@@ -75,8 +75,8 @@ decompression bomb         Safe             Safe              Safe             S
 2. :mod:`xml.dom.minidom` doesn't expand external entities and simply returns
    the unexpanded entity verbatim.
 3. :mod:`xmlrpclib` doesn't expand external entities and omits them.
-4. Since Python 3.8.0, external general entities are no longer processed by
-   default since Python.
+4. Since Python 3.8, external general entities are no longer processed by
```

Comment:
Should be `3.7.1` instead of `3.8`. And docs for 3.8 should refer to `3.7.1` too, because there were no vulnerable releases between 3.7.1 and 3.8.0.